include/lemlib/devices/encoder/rotation.hpp

```diff
@@ -20,7 +20,7 @@ class RotationEncoder : public Encoder {
          *
          * @return float angle rotated by the rotation sensor, in radians
```

Comment:
```suggestion
         * @return Angle angle rotated by the rotation sensor
```
update doxygen comment